src/controllers/scripting/controllerscriptenginebase.cpp

```diff
@@ -30,6 +30,11 @@ ControllerScriptEngineBase::ControllerScriptEngineBase(
     qRegisterMetaType<QMessageBox::StandardButton>("QMessageBox::StandardButton");
 }
 
+void ControllerScriptEngineBase::registerPlayerManager(
+        std::shared_ptr<PlayerManager> pPlayerManager) {
+    ControllerScriptEngineBase::s_pPlayerManager = pPlayerManager;
+}
```

Comment:
we should try to eliminate the static here. Would be be feasible to pass the PlayerManager in the `ControllerScriptEngineBase` constructor?